samples/apps/copilot-chat-app/deploy/main.json

```diff
@@ -4,8 +4,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.17.1.54307",
-      "templateHash": "14923066769528474387"
+      "version": "0.16.2.56959",
+      "templateHash": "18037485528098010448"
```

Comment:
Is this version downgrade intentional?

Reply:
It makes no difference.  It just lists the version of the Bicep to ARM JSON compiler used to generate this template.

As an aside, one can upgrade their bicep tool with this command:
az bicep upgrade

Reply:
Cool thx